runtime/runtime/tests/runtime_group_tools/random_config.rs

```diff
@@ -27,6 +27,8 @@ pub fn random_config() -> RuntimeConfig {
                 100,
             ),
             refund_gas_price_changes: !ProtocolFeature::ReducedGasRefunds.enabled(PROTOCOL_VERSION),
+            gas_refund_penalty: Rational32::new((rng.next_u32() % 100).try_into().unwrap(), 100),
```

Comment:
Why not generate a full range of possible `Rational32` values? That is,

```suggestion
            gas_refund_penalty: Rational::new(rng.next_u32(), u32::MAX),
```

Reply:
Charging anything outside of [0%,100%] would not make sense to configure.

Reply:
With `u32::MAX` in the denominator any u32 value (see also comment below on `Rational32`) in numerator would end up producing a rational in a `[0, 1.0]` range still. However unlike the current implementation it does have an opportunity to generate values like 99.9%, 99.99% and even 100% (which I believe the current implementation of this method would not generate due to `x % 100 < 100`.)

Reply:
Oh I see, yeah that makes sense then

Reply:
Changed to `gas_refund_penalty: Rational32::new(rng.gen_range(0..=i32::MAX), i32::MAX)`, keeping it a Rational32 bu generating the full range of positive parameters values